privacyidea/api/token.py

```diff
@@ -442,13 +441,15 @@ def unassign_api():
     You can either provide "serial" as an argument to unassign this very
     token or you can provide user and realm, to unassign all tokens of a user.
 
-    :return: In case of success it returns "value": True.
-    :rtype: json object
+    :return: In case of success it returns the number of unassigned tokens in "value".
+    :rtype: JSON object
     """
     user = get_user_from_param(request.all_data, optional)
     serial = getParam(request.all_data, "serial", optional)
+    g.audit_object.log({"serial": serial})
+
     res = unassign_token(serial, user=user)
-    g.audit_object.log({"success": True})
+    g.audit_object.log({"success": res > 0})
```

Comment:
By changing "True" to "res > 0" this would mean, if I assign "all tokens" of a user, who has not token, I would get a success=False.
Is that, what we want?

Reply:
I changed this to ``True`` again in dc564c7f3faaaf2ebc6315523cd0cdd481131c74 :)